holesky/README.md

```diff
@@ -87,6 +87,8 @@ Execute the following command based on the quorum you want to opt-in in EigenDA
 ```
 You only need to provide the quorum which you want to opt into. For example if you are already registered to quorum `0` and want to opt-in one more quorum `1`, then you just need to set `<quorum>` as `1` while opting in again.
 
+If you attempt to opt-in to both quorums ('`0,1`') you must have sufficient TVL to opt-in to the active Operator set for both quorums, otherwise the entire opt-in attempt will fail for both quorums. The opt-in attempt for both quorums is an "all or nothing" process.
```

Comment:
nit: is `Operator` intentionally capitalized in `... active Operator set...`?

Reply:
@ian-shim  Its a good point. @dabit3 raised the same question. Here's my preference on capitalization, but let me know if you feel differently

For now, my intention has been:

Lower case for generic verbs like "restake", "delegate(d)", etc.
Upper case to treat personas in our ecosystem as proper nouns (entities) like "Restaker", "Operator".
Upper case to refer to specific button labels like "click the Delegate button".

How do you like those rules?